narayana-release-process.sh

```diff
@@ -204,23 +199,27 @@ then
   exit
 fi
 git archive -o ../../narayana-full-$CURRENT-src.zip $CURRENT
-ant -f build-release-pkgs.xml -Drsync.host=${RSYNC_HOST} -Drsync.enabled=${RSYNC_ENABLED} -Dawestruct.executable="awestruct" all
```

Comment:
Maybe build-release-pkgs.xml needs to be deleted in this commit too?

Reply:
+1 , I will first double check all steps needed are well documented and then I will delete it (in this PR)

Reply:
Sorry @tomjenkinson , do you mean I should squash the two commits? If so I am planning to squash all commits at the end if that is ok with you.

Reply:
I am meaning something like `git rm build-release-pkgs.xml` from the root of the narayana checkout. Unless it has some use still?

Reply:
Thank you